Userland/Utilities/image.cpp

```diff
@@ -16,18 +16,37 @@
 #include <LibGfx/ImageFormats/QOIWriter.h>
 
 struct LoadedImage {
-    RefPtr<Gfx::Bitmap> bitmap;
+    Gfx::NaturalFrameFormat internal_format;
+    AK::Variant<RefPtr<Gfx::Bitmap>, RefPtr<Gfx::CMYKBitmap>> bitmap;
```

Comment:
Also, you shouldn't need to use `AK` here.

Reply:
VSCode's autocomplete adds it :( I thought I had removed them all, thanks for catching this.